### PR TITLE
feat: Init repo by kubebuilder

### PR DESCRIPTION
**Pull request by simon-cj**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
+/*
```
Comment (gaocegege) [inline main.go:1]: Please do not keep main.go in the root directory since we do not have only one CLI in the repo.

**Comment by simon-cj** [comment]
comments are addressed. PTAL

**Comment by simon-cj** [comment]
the first PR is too big, the main code is kubebuilder init and vendor.
PTAL.
@gaocegege 
/cc @ddysher

**Comment by gaocegege** [comment]
I think we can merge it. If there is any problem we can fix it later.

/lgtm

**Comment by simon-cj** [comment]
> Please help change the owners. Maybe add @codeflitting @simon-cj and @gaocegege as owners.

Owners is added

**Comment by ddysher** [comment]
/lgtm

LGTM to proceed.

**Pull request merged**